CHANGELOG.md

```diff
@@ -4,7 +4,7 @@
 
 TBD
 
-## [v2.0.0-alpha.1](https://github.com/medibloc/panacea-java/releases/tag/v2.0.0-alpha.1) - 2021-07-01 
+## [v2.0.0-alpha.2](https://github.com/medibloc/panacea-java/releases/tag/v2.0.0-alpha.2) - 2021-07-01 
```

Comment:
If the alpha version changes, I'm thinking about whether to specify it together or just one.

Reply:
I think your strategy is good enough. 
In the end, we will replace it with v2.0.0. 
So, please accumulate all PR titles.

Reply:
Oh Thank you. I writed bug docs to CHANGELOG.MD.